src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -890,6 +892,40 @@ class LeVADocumentUseCase extends DocGenUseCase {
         return this.createDocument(documentType, repo, data_, [:], modifier, null, watermarkText)
     }
 
+    String createTRC(Map repo = null, Map data = null) {
+        def documentType = DocumentType.TRC as String
+
+        def sections = this.jiraUseCase.getDocumentChapterData(documentType)
+        if (!sections) {
+            throw new RuntimeException("Error: unable to create ${documentType}. Could not obtain document chapter data from Jira.")
+        }
+
+        def systemRequirements = this.project.getSystemRequirements().collect { r ->
+            [
+                key: r.key,
+                name: r.name,
+                description: r.description,
+                risks: r.risks.join(", "),
+                tests: r.tests.join(", ")
+            ]
+        }
+
+        if (!sections."sec4") sections."sec4" = [:]
+        sections."sec4".systemRequirements = SortUtil.sortIssuesByProperties(systemRequirements, ["key"])
+
+        def data_ = [
+                metadata: this.getDocumentMetadata(this.DOCUMENT_TYPE_NAMES[documentType], repo),
+                data    : [
+                        sections: sections
```

Comment:
@martsec please use the vi editorconfig plugin https://github.com/editorconfig/editorconfig-vim :)